src/coprocessor/codec/mysql/duration.rs

```diff
@@ -457,6 +583,18 @@ mod tests {
             (b"00:00:00.777777", 2, Some("00:00:00.78")),
             (b"00:00:00.777777", 6, Some("00:00:00.777777")),
             (b"00:00:00.001", 3, Some("00:00:00.001")),
+            // NOTE:
+            (b"1:2:3", 0, Some("01:02:03")),
+            (b"1 1:2:3", 0, Some("25:02:03")),
+            (b"-.123", 3, Some("-00:00:00.123")),
+            (b"-", 0, Some("00:00:00")),
```

Comment:
According to MySQL this is invalid value:

```
mysql> select TIME '-';
ERROR 1525 (HY000): Incorrect TIME value: '-'
```

Reply:
OK, I will fix it.